docs/docs/migration-guide.mdx

```diff
@@ -594,9 +593,23 @@ policies:
   - name: TEDPolicy
     featurizer:
     - name: SingleStateFeaturizer
-
 ```
 
+### Class changes
+
+The following classes have moved from `rasa.core` to `rasa.shared.core`:
+
+| Rasa 1.0                             | Rasa 2.0                          |
+| ------------------------------------ | --------------------------------- |
+| rasa.core.events                     | rasa.shared.core.events           |
+| rasa.core.trackers                   | rasa.shared.core.trackers         |
+| rasa.core.channels.channel.RestInput | rasa.core.channels.rest.RestInput |
+| Rasa 1.0                             | Rasa 2.0                          |
+| ------------------------------------ | --------------------------------- |
+| rasa.core.events                     | rasa.shared.core.events           |
+| rasa.core.trackers                   | rasa.shared.core.trackers         |
+| rasa.core.channels.channel.RestInput | rasa.core.channels.rest.RestInput |
```

Comment:
duplicate
```suggestion
```

Reply:
This request came from a customer who was upgrading their bot to 2.0 and had to change their code.

Reply:
I just asked in [slack](https://rasacustomers.slack.com/archives/CSCRVPBBK/p1604080748017700)